invenio_rdm_records/records/mappings/os-v2/rdmrecords/records/record-v6.0.0.json

```diff
@@ -239,6 +239,301 @@
               },
               "default": {
                 "type": "keyword"
+              },
```

Comment:
should we bump version of the mapping? I know it is addition but the change is sizeable... WDYT?

Reply:
IMHO, I would prefer to keep the existing version since there are only backwards compatible additions and there was an effort to make it so.

I would also like to remove the old mapping versions, i.e. https://github.com/inveniosoftware/invenio-rdm-records/pull/1587